libres/tests/analysis/test_fwd_step_log.cpp

```diff
@@ -0,0 +1,44 @@
+#include <iostream>
+#include <string>
+#include <filesystem>
+
+#include "catch2/catch.hpp"
+
+#include "ert/analysis/fwd_step_log.hpp"
+
+class TmpLog {
```

Comment:
Wrap this class with an anonymous namespace: 
```c++
namespace {
class TmpLog {
    /* .. */
};
} // namespace
```

Anonymous namespaces are the official way to specify that a class is not to be exported out of the `.cpp` file (ie. private).